source/docs/software/getting-started/options-for-programming.rst

```diff
@@ -54,9 +55,12 @@ Advantages
 Disadvantages
 ^^^^^^^^^^^^^
 
-- Connecting to the robot controller's Wi-Fi network will prevent you from using anything on the internet including video tutorials and online communication
-- Using external libraries is difficult and borderline impossible
 - Steeper learning curve than Blocks
+- The OnBotJava editor lacks a lot of features (e.g. autocomplete) that make working with a text-based programming language convenient
+- Using external libraries is functionally impossible
```

Comment:
```suggestion
- Using external libraries is more difficult than in Android Studio
```

Reply:
If there's a way to use external libraries on OnBotJava that isn't functionally impossible, we should absolutely document that process.  However, I don't know of such a process, and I haven't seen anyone else talking about such a process existing either.

This will change in the 2027-28 season anyways, with SystemCore having a much more competent and useful OnBot environment based on VSCode.